src/util/errorHandler.ts

```diff
@@ -0,0 +1,10 @@
+export function errorHandler(err) {
+    const { status, error } = err.response.data
+
+    if (status && error) {
+        throw new Error(`${status} ${error}`)
```

Comment:
Do we want to throw something like a `MythXAPIError` (We can probably figure out a better name...) here so that it is clear where the error occurs and it is more easily caught?